HTML/task5/index.html

```diff
@@ -0,0 +1,109 @@
+        <label class='label__font' for='18px'>Bigger font-size</label>
+        <input type='checkbox' id='dark-mode' />
+        <label class='label__font' for='dark-mode'>Dark mode</label>
+        <section class='container'>
+            <article class="container__article container__article--full">
+                <h2>Grumpy cat is dead</h2>
+                <p> San Francisco (CNN Business)Grumpy Cat — the angry-looking celebrity feline who launched countless
+                    internet memes — died on Tuesday. The beloved cat leaves behind her owner Tabatha Bundesen, a small
+                    empire of books and merchandise bearing her unique face, movie and TV appearances, and millions of
+                    Google hits.
+                    Grumpy Cat is also likely to live on in one particular area of technology: artificial intelligence.
+                    Images spread across the internet are often collected by researchers and used to train artificial
+                    intelligence algorithms to do tasks like recognize people or animals in images, or even learn how to
+                    generate wholly new images of people or animals.
+                </p>
+                <figure>
+                    <img class='article__img' src='img/task5-grumpycat.jpg' alt='Grumpy cat'>
+                    <figcaption>Grumpy cat during the last press conference</figcaption>
+                </figure>
+            </article>
+            <article>
+                <h2>Fattest cat</h2>
+                <p> In previous editions of the Guinness World Records book, the editors recorded heaviest animals and
+                    pets.
+                    The heaviest cat of all, for example, was Himmy, owned by Thomas Vyse (Australia), who weighed
+                    21.3kg
+                    (46lb 15 ½ oz) when it died on 12th March 1986 at the age of 10 years 4 months. </p>
+                <figure>
+                    <img class='article__img' src='img/task5-grumpycat.jpg' alt='Fat cat'>
+                    <figcaption>Himmy - the fattest cat</figcaption>
+                </figure>
+                <p>Himmy – who had to be transported in a wheelbarrow – was thankfully unchallenged until 1998, when
+                    editors
+                    and record managers decided to discontinue the category to deter people from over-feeding their pets
+                    just to appear in the book. Today, we still monitor animal weight and size by breed – such as the
+                    largest breed of horse – but not individual specimens.
+                </p>
+
+            </article>
+            <article>
+                <h2>Most views for a cat on YouTube</h2>
+                <p> Full of character and sublimely lazy, Maru is the most viewed cat on YouTube. His name means “round”
+                    in
+                    Japanese, and it’s partly his tubbiness that has won the Scottish fold cat so many fans.</p>
```

Comment:
it is a bit better to write things like this in following way, as it is a bit easier to read code when content separated from tags
```html
<p>
    Full of character and sublimely lazy, Maru is the most viewed cat on YouTube. His name means “round” in
    Japanese, and it’s partly his tubbiness that has won the Scottish fold cat so many fans.
</p>
```